src/config.rs

```diff
@@ -331,7 +331,19 @@ fn merge_configuration_files(
 
             for (variable_name, variable_value) in package.variables {
                 if first_package.variables.contains_key(&variable_name) {
-                    anyhow::bail!("variable {:?} already encountered", variable_name);
+                    // Panic safety: Already checked via `contains_key` above
+                    let mut first_value = first_package.variables.get_mut(&variable_name).unwrap();
+                    match (&mut first_value, &variable_value) {
+                        (toml::Value::Table(_a), toml::Value::Table(_b)) => {
+                            info!("Merging {:?} tables", variable_name);
```

Comment:
this should be `trace!`, not `info!` since it shouldn't appear with just a single `-v`, it fits more into `-vvv` category.